src/UNIV2LPOracle.sol

```diff
@@ -84,11 +93,18 @@ contract UNIV2LPOracleFactory {
     event NewUNIV2LPOracle(address sender, address orcl, bytes32 wat, address indexed tok0, address indexed tok1, address orb0, address orb1);
```

Comment:
Should probably change the name of the `sender` argument to `owner` and pass the value appropriately in `build`.

Reply:
done